server/handlers.go

```diff
@@ -189,7 +189,8 @@ func (s *Server) VerifyChallenge(w http.ResponseWriter, r *http.Request) {
 	sliceKey := bloomKey(iatTime)
 
 	if err := s.ensureBloom(ctx, sliceKey); err != nil {
-		http.Error(w, "bloom filter initialization error: "+err.Error(), http.StatusInternalServerError)
+		log.Printf("bloom filter initialization error: " + err.Error())
```

Comment:
use `log.Printf("bloom filter initialization error: %v", err)` like on the untouched line 201